ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 5.6.1f1
+m_EditorVersion: 2017.1.0f3
```

Comment:
Unity claims the entire Project Settings folder is necessary. Perhaps this particular file is not necessary but they don't mention it just to be safe (because deleting anything else from here would break things). But it's also 28 bytes, so we're probably fine including it lol